bin/ream/src/main.rs

```diff
@@ -144,9 +145,17 @@ pub async fn run_lean_node(config: LeanNodeConfig, executor: ReamExecutor) {
     // TODO 1: Load keystores from the config.
     // TODO 2: Add RPC service for lean node.
     let chain_service = LeanChainService::new(lean_chain.clone(), chain_receiver).await;
+    let gossipsub_config = LeanGossipsubConfig::default();
+    let discovery_config = LeanDiscoveryConfig::new(
```

Comment:
why is this called a discovery config, these variables should just be apart of the already existing LeanNetworkConfig, as the socket ip/port are just network configurations not Discv5 configurations